guard/templates/deployment.yaml

```diff
@@ -82,7 +79,7 @@ spec:
         operator: Exists
       {{- end }}
       {{- if .Values.tolerations }}
-{{ toYaml .Values.extraTolerations | indent 8 }}
+{{ toYaml .Values.tolerations | indent 8 }}
```

Comment:
Since `tolerations ` and `extraTolerations` existed, I sent it to `tolerations`.